core/src/exampm.cc

```diff
@@ -0,0 +1,106 @@
+//---------------------------------------------------------------------------//
```

Comment:
Is this file for a new example problem? It may be better to add it as an example instead.

Reply:
I honestly don't know remember creating this file. I will just delete it.